package.yaml

```diff
@@ -81,6 +91,10 @@ library:
     - Pairing.Pairing
     - Pairing.Jacobian
     - Pairing.CyclicGroup
+    - Pairing.Hash
+
+  other-modules:
+    - Pairing.Modular
```

Comment:
Any reason not to expose this module?

Reply:
Apart from me not expecting to use those functions outside of the Fq, Group and Hash modules, not really. Do you see it being used outside of the pairing library? If so happy to move it out.